src/buildblock/ProjDataInfoBlocksOnCylindricalNoArcCorr.cxx

```diff
@@ -120,8 +120,8 @@ find_scanner_coordinates_given_cartesian_coordinates(int& det1, int& det2, int&
 
   DetectionPosition<> det_pos1;
   DetectionPosition<> det_pos2;
-  if (get_scanner_ptr()->find_detection_position_given_cartesian_coordinate(det_pos1, c1-this->z_shift)==Succeeded::no ||
-      get_scanner_ptr()->find_detection_position_given_cartesian_coordinate(det_pos2, c2-this->z_shift)==Succeeded::no)
+  if (get_scanner_ptr()->find_detection_position_given_cartesian_coordinate(det_pos1, c1+this->z_shift)==Succeeded::no ||
+      get_scanner_ptr()->find_detection_position_given_cartesian_coordinate(det_pos2, c2+this->z_shift)==Succeeded::no)
```

Comment:
because for the test the ax_pos for the shift was calculated on the last ring instead of the first and I had based the shift on the test but for direct sino the point A and B of the LOR have the same z which is negative